### ISSUE TITLE
Support jfrog artifactory

### ISSUE THREAD

**Issue by tu1h** [vmware-tanzu/asset-relocation-tool-for-kubernetes#192]
I have a docker registry repository addressed as `172.30.41.0:8091/docker` and a helm repository addressed as `http://172.30.41.0:8091/artifactory/helm-local` on Jfrog Artifactory.

But I got an error like this `credentials server name "172.30.41.0:8091/docker" must not contain a scheme ("http://") nor a path. Example of valid registry names
are "myregistry.io" or "myregistry.io:9999"`

**Comment by frizzr** [comment]
Yes, this is a **very** large limitation to this tool. Hopefully it is a relatively easy fix... -fingers crossed-
